doc/source/whatsnew/v0.25.0.rst

```diff
@@ -305,7 +305,7 @@ Categorical
 ^^^^^^^^^^^
 
 - Bug in :func:`DataFrame.at` and :func:`Series.at` that would raise exception if the index was a :class:`CategoricalIndex` (:issue:`20629`)
--
+- Bug in :func:`_cat_compare_op` that would valuate comparison with None to True (:issue:`26504`)
```

Comment:
Our `whatsnew` entries try to describe the bugs from the perspective of user-facing functionality.  In this case, the bug is affecting comparisons between `Categorical` and scalars.  Can you update the entry to describe what's being affected there instead?

Reply:
Thank you for the suggestion! Will do!